tools/apidoc/gen_toc.py

```diff
@@ -190,7 +190,8 @@
     'CA': 'Certificate',
     'listElastistorInterface': 'Misc',
     'cloudian': 'Cloudian',
-    'Sioc' : 'Sioc'
+    'Sioc' : 'Sioc',
+    'remoteDiganostics': 'Virtual Machine'
```

Comment:
Rename as `'Diagnostics': 'Diagnostics'`. When you build cloudstack, go to tools/apidocs/target and open the generated html to see how APIs are grouped in sections.

Reply:
Resolved